dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedFileReader.java

```diff
@@ -87,14 +94,60 @@ public static PagedFileReader create(AlluxioConfiguration conf, CacheManager cac
    * @param startPosition
    */
   public PagedFileReader(CloseableResource<UnderFileSystem> ufs,
```

Comment:
i think for this file, we mainly use the transferTo method?
trying to figure out what's the call stack from worker side that leverage the zero-copy

Reply:
`DataFileChannel#getNettyOutput()` will return a `DefaultFileRegion` object that represents a file region to be read. Netty will call `DefaultFileRegion#transferTo` when we write a `DefaultFileRegion` object to the channel. So what I do in the PR is to return a `List` of `DefaultFileRegion` for `RPCMessageEncoder`.